src/saltext/vmware/utils/connect.py

```diff
@@ -78,26 +77,38 @@ def get_service_instance(opts=None, pillar=None, esxi_host=None):
                     user: admin
                     password: ***
 
+    If configuration for multiple VMWare services instances is required, they can be
+    set up as different configuration profiles:
+    For example:
+    .. code-block:: yaml
+
+        vmware_config:
```

Comment:
**suggestion** this would become `saltext.vmware:`

Reply:
Done